src/compas_ghpython/artists/meshartist.py

```diff
@@ -31,6 +29,25 @@ def __init__(self, mesh, **kwargs):
     def draw(self, color=None):
         """Draw the mesh as a RhinoMesh.
 
+        Parameters
+        ----------
+        color : tuple, optional
+            The color of the mesh.
+            Default is the value of ``~MeshArtist.default_color``.
+
+        Returns
+        -------
+        :class:`Rhino.Geometry.Mesh`
+        """
+        color = color or self.default_color
+        vertices, faces = self.mesh.to_vertices_and_faces()
+        return compas_ghpython.draw_mesh(vertices, faces, color)
+
+    def draw_mesh(self, color=None):
```

Comment:
I updated it to match rhino & blender's behavior. We can discuss whether this is a default we want to change in the future.

I tend to favor a change because `draw` -intuitively to me- should draw the "entity" at hand (ie. mesh in this case), and not its constituting parts. But let's defer the conversation for later.